### DISCUSSION TITLE
COMP2021-OBJECT-ORIENTED PROGRAMMING-评价待收集

### DISCUSSION THREAD

**Discussion by WPCJATH** [ZHANG-CAIQI/PolyU_COMP#63] [课程评价]
评论请带上课时间(e.g., 21/22 sem1)或老师(e.g., Chen Dawen Peter, 陈大文, Peter)，谢谢！

**Comment by elleEnjolras** [comment]
workload大
认真听课是王道
lab的内容非常可能在作业中出现
